worldguard-core/src/main/java/com/sk89q/worldguard/session/Session.java

```diff
@@ -103,6 +103,8 @@ public void initialize(LocalPlayer player) {
 
         lastValid = location;
         lastRegionSet = set.getRegions();
+        disableBypass = WorldGuard.getInstance().getPlatform().getGlobalStateManager()
+                .get(player.getWorld()).disableDefaultBypass;
 
```

Comment:
Would like to have something here like `if (disableBypass) player.printInfo("You are bypassing region protection...")` so the user knows.

Reply:
DisableBypass means you're not bypassing region protections. Your suggested message doesn't make sense

And imo that shouldn't be necessary or another config option. On many servers there's already a spam at login from different plugins, I don't like the idea to add another message to the login spam

Reply:
I added a configurable message for users who can toggle bypass.